clang/test/CodeGen/cheri/riscv/non-cheri-march.c

```diff
@@ -0,0 +1,11 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py
+/// Using __capability without the appropriate -march string previously
+/// crashed clang during codegen instead of emitting an error
+// RUN: %clang_cc1 -triple riscv64 -o - -emit-llvm %s -verify=no-cheri
+// RUN: %clang_cc1 -triple riscv64 -target-feature +xcheri -o - -emit-llvm %s | FileCheck %s
```

Comment:
Not sure what test coverage this gives us when we have umpteen CHERI CodeGen tests elsewhere that would fail if this didn't work?

Reply:
Makes sense - will drop after moving to parser.